wcivf/apps/elections/templates/elections/includes/_single_ballot.html

```diff
@@ -51,11 +51,11 @@ <h2>{% if postelection.is_london_assembly_additional %}Additional members{% else
             {% if postelection.voting_system.slug == "PR-CL" %}
               You will have one vote, and can vote for a single party list or independent candidate.
             {% else %}
-              You can choose from <strong>{{ postelection.party_ballot_count }}</strong>
-              in the {{ postelection.friendly_name }}.
               {% if postelection.winner_count and postelection.election.voting_system_id == 'FPTP' %}
-                You will have {{ postelection.winner_count|intword }} vote{{ postelection.winner_count|pluralize }}.
+                You will have up to {{ postelection.winner_count|intword }} vote{{ postelection.winner_count|pluralize }}.
               {% endif %}
+              You can choose from <strong>{{ postelection.party_ballot_count }}</strong>
+              in the {{ postelection.friendly_name }}.
```

Comment:
Struggling to follow the nested if statements a little but it looks like this line is always displayed, is that intended? So if it was FTPT you would get both this and the lines on 55.

Reply:
Yes, approved by @pmk01